docs/reference/mapping/runtime.asciidoc

```diff
@@ -708,39 +706,50 @@ The mapping contains two fields: `@timestamp` and `message`.
 ----
 // TESTRESPONSE[s/\.\.\./"settings": $body.my-index.settings/]
 
-[[runtime-examples-runtime-field]]
-==== Define a runtime field to search by IP address
-If you want to retrieve results that include `clientip`, you can add that field
-as a runtime field in the mapping. The runtime script operates on the `clientip`
-field at runtime to calculate values for that field.
+[[runtime-examples-grok]]
+==== Define a runtime field with a grok pattern
+If you want to retrieve results that include `clientip`, you can add that
+field as a runtime field in the mapping. The following runtime script uses
+the grok operator, which extracts structured fields out of a single text
```

Comment:
nit: `operator` is a technical term in language design for things like `+`, `-`, `?.`, and `:?`. `grok` isn't an operator. It's probably more correct just to the "this script defines a `grok` pattern and then uses it to extract the contents of the log line.